pkg/services/authz/server.go

```diff
@@ -51,14 +53,18 @@ func (s *legacyServer) Read(ctx context.Context, req *authzv1.ReadRequest) (*aut
 
 	action := req.GetAction()
 	subject := req.GetSubject()
-	stackID := req.GetStackId() // TODO can we consider the stackID as the orgID?
+	namespace := req.GetNamespace() // TODO can we consider the stackID as the orgID?
```

Comment:
This one I'm not sure. What's a format now? Do we have a special helper function to extract orgId from name space?

Reply:
You can use the authlib claims to parse this:
```suggestion
	namespace := req.GetNamespace() 
	
	info, err := claims.ParseNamespace(namespace)
	if err != nil || info.OrgID == 0 {
		return "", fmt.Errorf("invalid namespace: %s", namespace)
	}
```